homeassistant/components/recorder/models.py

```diff
@@ -61,7 +61,7 @@ class States(Base):   # type: ignore
     domain = Column(String(64))
     entity_id = Column(String(255))
     state = Column(String(255))
-    attributes = Column(Text)
+    attributes = Column(Text, default='{}')
```

Comment:
You will need to write a migration if we are going to change the schema - otherwise there will be two different versions of the schema around. Safer to restore the old way of doing it.